desktop/lib/middleware/hsts.coffee

```diff
@@ -1,14 +0,0 @@
-#
-# HSTS allows for a more effective implementation of TLS by ensuring
```

Comment:
This diff makes it look like files are being deleted, but it's just middleware being de-duplicated between desktop/mobile as you can see from [this line](https://github.com/artsy/force/pull/1197/files#diff-59f3431023761d7afa724572082feebb). Tests have also been moved up to ./test/lib.